micah-akpan/questioner-react
# chore(Project setup): finish setting up project

#### What does this PR do?
Setup Project codebase structure
#### Description of Task to be completed?
Setup Project
#### How should this be manually tested?
- Clone repo
- Install dependencies: `npm install`
- Run `npm run dev:start`
- You should be greeted with a nice hello message in your browser
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#164695740
#### Screenshots (if appropriate)
#### Questions:
